scrc/enums/section.py

```diff
@@ -13,4 +13,9 @@ class Section(Enum):
 
     @classmethod
     def without_facts(cls):
-        return cls.HEADER, cls.HEADER,cls.CONSIDERATIONS, cls.RULINGS, cls.FOOTER
\ No newline at end of file
+        return cls.HEADER, cls.HEADER,cls.CONSIDERATIONS, cls.RULINGS, cls.FOOTER
+    
+    @classmethod
+    def without_topic(cls):
+        return cls.FULLTEXT, cls.HEADER, cls.FACTS, cls.CONSIDERATIONS, cls.RULINGS, cls.FOOTER
+    
```

Comment:
Would it make sense to refactor it so that it takes every section and then just the one in question (facts, topic) is removed?

Reply:
I changed it so topic will be included.